cmd/subnetcmd/deploy.go

```diff
@@ -256,13 +258,131 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 	return app.UpdateSidecar(&sidecar)
 }
 
-func getControlKeys(network models.Network) ([]string, bool, error) {
+func getControlKeys(network models.Network, keyName string) ([]string, bool, error) {
 	controlKeysInitialPrompt := "Configure which addresses may add new validators to the subnet.\n" +
 		"These addresses are known as your control keys. You will also\n" +
-		"set how many control keys are required to add a validator."
-	controlKeysPrompt := "Set control keys"
+		"set how many control keys are required to add a validator (the threshold)."
+	moreKeysPrompt := "P-Chain addresses from your private key will be added by default to the control key set. Would you like to add more?"
```

Comment:
I think we should change this prompt. Instead of saying "We're adding all keys, add more? No/Yes" we should do "How would you like to set your control keys?" 1. Use all stored keys 2. Use creation key only 3. Custom list (that does not automatically include the stored keys)

Reply:
Updated according to this request.